tests/browsercontext-proxy.spec.ts

```diff
@@ -14,9 +14,24 @@
  * limitations under the License.
  */
 
-import { browserTest as it, expect } from './config/browserTest';
-
-it.use({ proxy: { server: 'per-context' } });
+import os from 'os';
+import { Browser } from '..';
+import { browserTest, expect } from './config/browserTest';
+
+const LAUNCH_OPTION_OVERRIDES = os.platform() === 'win32' ? { proxy: { server: `per-context` } } : {};
+
+const it = browserTest.extend<{ browserTestFactory: () => Promise<Browser> }>({
```

Comment:
Looks like you added it twice. Also, can we start a single browser for these tests to save time? A worker-level browserWithProxy for windows that is...

While you are at it, line #40 can be fixed.